zuul.d/molecule.yaml

```diff
@@ -191,3 +191,14 @@
       - ^molecule-requirements.txt
       - ^ci_framework/roles/test_deps/(?!meta).*
       - ^ci/playbooks/molecule.*
+- job:
+    name: cifmw-molecule-openshift_setup
+    nodeset: centos-9-crc-xl
+    parent: cifmw-molecule-base
+    vars:
+      TEST_RUN: openshift_setup
+    files:
+      - ^ansible-requirements.txt
+      - ^molecule-requirements.txt
+      - ^ci_framework/roles/openshift_setup/(?!meta).*
```

Comment:
Based on this pr ttps://github.com/openstack-k8s-operators/ci-framework/pull/195: the above line needs to be update with 
```
^ci_framework/roles/openshift_setup/(?!meta|README).*
```

Reply:
indeed, good catch! A rebase + running `make role_molecule` would re-generate it right.